### DISCUSSION TITLE
Problem installing and using ipykernel for jupyter

### DISCUSSION THREAD

**Discussion by Theoflo** [microsoft/vscode-jupyter#15248] [Questions and Answers]
Hi, I'm trying to open a document using Jupyter on Visual Studio Code but when I try launching the first request I'm asked to install ipykernel. And when I do that (using 'pip install ipykernel' or 'pip install ipykernel -U --user --force-reinstall') after some steps an error message telling me that the file _implementation.pyi in directory          Packages\\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python311\\site-packages\\jedi\\third_party\\typeshed\\third_party\\2and3\\requests\\packages\\urllib3\\packages\\ssl_match_hostname                     does not exist.

Can you help me, I've already try uninstalling and reinstalling python, Jupyter, VSC, and it seems that nothing is working...

**Comment by DonJayamanne** [comment]
> SoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\jedi\third_party\typeshed\third_party\2and3\requests\packages\urllib3\packages\ssl_match_hostname does not exist

Thanks for posting here, and sorry you are running into this.
Looks like its an error in Python itself,
I'd suggest deleting all of these folders and then re-installing Python.

Else, I'd suggest trying stack overflow.

**Comment by DonJayamanne** [comment]
Re-opening in case someone else can provide a solution
